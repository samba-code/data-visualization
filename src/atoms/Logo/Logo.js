import React from "react";
import LogoImage from "./samba-code-logo-white-2x.png";
import Obfuscate from "react-obfuscate";

const Logo = () => (
  <div>
    <Obfuscate email="hello@sambacode.net" aria-label="Email Samba Code">
      <img width={150} height={30} src={LogoImage} alt="Samba Code" />
    </Obfuscate>
  </div>
);

export default Logo;
