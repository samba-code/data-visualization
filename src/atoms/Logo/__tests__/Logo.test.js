import React from "react";
import { render } from "enzyme";

import Logo from "../Logo";

describe("Logo", () => {
  it("should match snapshot", () => {
    const wrapper = render(<Logo />);
    expect(wrapper).toMatchSnapshot();
  });
});
