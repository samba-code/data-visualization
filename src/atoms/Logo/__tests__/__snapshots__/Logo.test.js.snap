// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should match snapshot 1`] = `
<div>
  <a
    aria-label="Email Samba Code"
    href="obfuscated"
    style="unicode-bidi:bidi-override;direction:rtl"
  >
    <img
      alt="Samba Code"
      height="30"
      src="[object Object]"
      width="150"
    />
  </a>
</div>
`;
